frontend/src/Todo/Header/Header.jsx

```diff
@@ -0,0 +1,42 @@
+import { useContext, useState } from 'react';
+import classes from './Header.module.css';
+import { Context } from '../Context';
+
+export default function TodoHeader() 
+{
+	const { container, input, btn } = classes;
+
+	const { addTodo } = useContext(Context);
+
+	const [ todoName, setTodoName ] = useState('');
+
+	const onSubmitHandler = e => 
+	{
+    e.preventDefault();
+    if (todoName) 
+    {
+    	const id = new Date().valueOf();
+    	const todoItem = {
+        id,
+        isCompleted: false,
+        name: todoName
+      };
+
+    	addTodo(todoItem);
+    	setTodoName('');
+    }
```

Comment:
Wrong indent, you may be confused with assign id like this. If you are using mongoDb, mogno assign id itself on every todo.